From 653f4266a349d0b2f3de3e1b1ca0a6774935564d Mon Sep 17 00:00:00 2001
From: Joseph C Wang <joequant@gmail.com>
Date: Sun, 20 Nov 2022 03:41:17 +0800
Subject: [PATCH] add missing gmodule depends

---
 libxapp/meson.build | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libxapp/meson.build b/libxapp/meson.build
index 62bd56d..0005347 100644
--- a/libxapp/meson.build
+++ b/libxapp/meson.build
@@ -4,12 +4,14 @@ gio_dep = dependency('gio-2.0', version: glib_min_ver, required: true)
 gio_unix_dep = dependency('gio-unix-2.0', version: glib_min_ver, required: true)
 glib_dep = dependency('glib-2.0', version: glib_min_ver, required: true)
 gtk3_dep = dependency('gtk+-3.0', version: '>=3.16', required: true)
+gmodule_dep = dependency('gmodule-2.0', version: glib_min_ver, required: true)
 
 libdeps = []
 libdeps += gio_dep
 libdeps += gio_unix_dep
 libdeps += glib_dep
 libdeps += gtk3_dep
+libdeps += gmodule_dep
 libdeps += dependency('gdk-pixbuf-2.0', version: '>=2.22.0', required: true)
 libdeps += dependency('cairo', required: true)
 libdeps += dependency('x11', required: true)
